README.md

```diff
@@ -165,6 +165,25 @@ Take the following steps to launch applications from sdl core.
   6. Both applications should show up on the head unit for launching
   7. Select the other application, and you should see it launched and brought to the foreground on the phone
 
+### Test Covarage
+Used technologies :
+  * GCOV - is a test coverage program.
+  * LCOV - is a graphical front-end for GCC's coverage testing tool for gcov.
+  * codecov.io - is service for assembling code coverage and represent in form clear for reading.
```

Comment:
@Kozoriz, codecov desription looks not good to me. I think it should be converted to something like this
"service for assembling code coverage and representing it in a clear for reading form."

Also in my opinion "is a" is redundant in the prev two descriptions since you wrote "-" there. So the desc can look like "GCOV - test ..."